doc/classes/EditorPlugin.xml

```diff
@@ -471,6 +471,7 @@
 			<return type="int">
 			</return>
 			<description>
+				Updates the overlays of the [2D/3D] viewport.
```

Comment:
I'm not sure if the square brackets will work here, I'd need to build the docs to check. Usually they are part of the markup. If you're sure this works, ignore me, if not, maybe change to "(2D/3D)" or escape the square brackets with a backslash?

Reply:
I would change it to "(2D/3D)". thanks. :)